spec/controllers/admin/refund_queues_controller_spec.rb

```diff
@@ -30,10 +35,23 @@
       props = inertia.props
       expect(props[:title]).to eq("Refund queue")
       expect(props[:users]).to match_array([
-                                             hash_including(id: users[0].id),
-                                             hash_including(id: users[1].id),
-                                             hash_including(id: users[2].id)
+                                             hash_including(id: user_1.id),
+                                             hash_including(id: user_2.id),
+                                             hash_including(id: user_3.id)
                                            ])
     end
+
+    it "renders the page with pagination" do
+      get :show, params: { page: 2, per_page: 2 }
+
+      expect(response).to be_successful
+      expect(inertia.component).to eq "Admin/RefundQueues/Show"
+
+      props = inertia.props
+      expect(props[:pagination]).to be_present
+      expect(props[:users]).not_to match_array([hash_including(id: user_1.id)])
+      expect(props[:users]).not_to match_array([hash_including(id: user_2.id)])
+      expect(props[:users]).to match_array([hash_including(id: user_3.id)])
```

Comment:
```suggestion
      expect(props[:users]).to contain_exactly(hash_including(id: user_3.id))
```

AFAIK `match_array` asserts that only the given element(s) are present, so we don't need the other checks. `contain_exactly` is essentially equivalent minus the array brackets and makes the expectation clearer to me